data/mundo/cpsAssets/23263889.json

```diff
@@ -384,7 +384,7 @@
       {
         "required": false,
         "tile": "Include from VisJo",
-        "href": "/include/vjamericas/176-eclipse-lookup/mundo/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-werty/venta-app_eclipse-mundo_bjd2w.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-werty/venta-app_eclipse-mundo_bjd2w.png",
+        "href": "/include/vjamericas/176-eclipse-lookup/mundo/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-werty/venta-app_eclipse-mundo_bjd2w.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640",
```

Comment:
Why has this fallback image been removed?

Reply:
Hmm not sure why, I'll add it back and see if it affects anything

Reply:
Did you work this out?

Reply:
Looks like it was just taken out for testing and accidentally committed

Reply:
It might be better to keep this amp image fallback removed, as it gives a tester asset to test the fallback for